lib/constants.js

```diff
@@ -1,20 +1,15 @@
 'use babel'
 
-export const MachineStatusText = [
-  null,
-  'Offline',
-  'Online',
-  'Connected'
-]
+export const MachineStatusText = [null, 'Offline', 'Online', 'Connected']
 
 export const MachineStatus = {
   Offline: 1,
   Online: 2,
-  Connected: 3
+  Connected: 3,
```

Comment:
I know it's not, I can see how it can make it easier. That's only my opinionated view 💀